### PR TITLE
Add CODEOWNERS file

### PR DESCRIPTION
**Pull request by lmontrieux**
# Add CODEOWNERS file

## Description
A CODEOWNERS file lists who is able to merge to master. This simple CODEOWNERS file allows anyone in @zalando/team-nakadi to merge anything to master.

### Why now?
Since the open source team has made changes to the teams structure in the Zalando org on github.com, we need to use the github review mechanism to review and approve PRs.
Currently, all team members are admin, and can bypass the restrictions to merge to master. This will eventually go away, and the only way to merge to master will be to be in the CODEOWNERS file.

### CODE REVIEW

**Comment by antban** [comment]
:+1:

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+*   @zalando/team-nakadi
```
Comment (adyach) [inline .github/CODEOWNERS]: no new line

Comment (lmontrieux) [reply .github/CODEOWNERS]: 43be24b

**Comment by adyach** [comment]
👍

**Comment by lmontrieux** [comment]
👍

**Pull request merged**